monetio/models/cmaq.py

```diff
@@ -115,7 +115,7 @@ def open_mfdataset(
     """
 
     # open the dataset using xarray
-    dset = xr.open_mfdataset(fname, concat_dim="TSTEP", **kwargs)
+    dset = xr.open_mfdataset(fname, combine='nested', concat_dim="TSTEP", **kwargs)
```

Comment:
```suggestion
    dset = xr.open_mfdataset(fname, combine="nested", concat_dim="TSTEP", **kwargs)
```
linting should pass with double quotes here

Reply:
I know you marked this resolved, but we do need this, so I'm just going to commit it.